packages/@snowpack/app-scripts-lit-element/snowpack.config.js

```diff
@@ -1,35 +1,15 @@
-const fs = require("fs");
-const path = require("path");
+const fs = require('fs');
+const path = require('path');
 
 const cwd = process.cwd();
-const isTS = fs.existsSync(path.join(cwd, "tsconfig.json"));
-
-const scripts = {
-  "mount:public": "mount public --to /",
-  "mount:src": "mount src --to /_dist_",
-};
-
-if (isTS) {
-  scripts["run:tsc"] = "tsc --noEmit";
-  scripts["run:tsc::watch"] = "$1 --watch";
-}
-
-const buildId = isTS ? "build:js,ts" : "build:js";
-if (
-  fs.existsSync(path.join(cwd, "babel.config.json")) ||
-  fs.existsSync(path.join(cwd, "babel.config.js")) ||
-  fs.existsSync(path.join(cwd, "babel.config.cjs")) ||
-  fs.existsSync(path.join(cwd, "babel.config.mjs"))
-) {
-  scripts[buildId] = "babel --filename $FILE";
-} else {
-  const bundledConfig = path.join(__dirname, "babel.config.json");
-  scripts[buildId] = `babel --filename $FILE --config-file ${bundledConfig}`;
-}
+const isTS = fs.existsSync(path.join(cwd, 'tsconfig.json'));
 
 module.exports = {
-  scripts,
-  plugins: ["@snowpack/plugin-dotenv"],
+  mount: {
+    public: '/',
+    src: '/_dist_',
+  },
+  plugins: ['@snowpack/plugin-babel', '@snowpack/plugin-dotenv'],
```

Comment:
Probably the biggest template change:

1. This uses `@snowpack/plugin-babel` rather than a run script
1. It also moves the `babel.config.json` into the template itself. I know it’s one more piece of configuration, but if a user needed to customize anything at all, we made it really difficult for them. This change lets us manage less, and gives users more control.